specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-01-20-preview/guestconfiguration.json

```diff
@@ -309,6 +309,100 @@
           "application/json"
         ]
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/providers/Microsoft.GuestConfiguration/software/{softwareId}": {
```

Comment:
>subscriptions [](start = 6, length = 13)

Any specific reason to have in on an old version of the api instead of 2018-06-30?

Reply:
Fixed, to use newer version